src/libmeasurement_kit/nettests/captive_portal.cpp

```diff
@@ -0,0 +1,25 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include "private/nettests/runnable.hpp"
+
+#include <measurement_kit/nettests.hpp>
+#include <measurement_kit/ooni.hpp>
+
+namespace mk {
+namespace nettests {
+
+CaptivePortalTest::CaptivePortalTest() : BaseTest() {
+    runnable.reset(new CaptivePortalRunnable);
+    runnable->test_name = "captive_portal";
```

Comment:
According to the spec the test name is actually `captiveportal` written altogether (https://github.com/TheTorProject/ooni-spec/pull/87/files#diff-2474b11b8ed0dee978a9293a881a3c73R185)